pkg/webhook/configuration.go

```diff
@@ -210,7 +214,8 @@ func (c *client) createAdmissionValidatingConfig(
 		// SideEffects:             &sideEffectClass,
 		// AdmissionReviewVersions: []string{"v1"},
 		TimeoutSeconds: &five,
-		FailurePolicy:  &Fail,
+		//FailurePolicy:  &Fail,
+		FailurePolicy: failurePolicy(),
```

Comment:
If the user set policy as `Ignore` in env and upgraded to 1.12.0 thinking they marked `FailurePolicy` as ignore but it is not the case.

Reply:
No, they have to delete the valdatingwebhookconfiguration and set the ENV which will restart the admission pod and config will be created again.

Reply:
As part of upgrade -- can't we take this part also into account and upgrade job should/can handle this. @mittachaitu @prateekpandey14

Reply:
n_ : remove the commented line. 

Also, as this was in alpha, we are ok to make this change.

Reply:
commented line removed